charmcraft.yaml

```diff
@@ -6,3 +6,7 @@ bases:
     run-on:
     - name: "ubuntu"
       channel: "20.04"
+parts:
```

Comment:
This producing the error
```
Bad charmcraft.yaml content:
- extra field 'charm' not permitted in 'parts' configuration (full execution logs in '/home/balbir/snap/charmcraft/common/charmcraft-log-1lw3qw7d')
```
I am using latest stable charmcraft version 1.1.1.

Reply:
Yeah you will need to update to 1.2.0, it is pretty unavoidable :-)